Binary Converter/js/formHandler.js

```diff
@@ -0,0 +1,15 @@
+let form = document.querySelector('form');
+
+form.onsubmit = () => {
+    let number = document.querySelector('#number').value.toLowerCase().split('');;
+    let from = document.querySelector('#from').value;
+    let to = document.querySelector('#to').value;
+
+    const numberHandler = new BinaryConverter();
+    
+
+    document.querySelector('#answer').textContent = numberHandler.convert(number, from, to).toUpperCase();
+    return false;
```

Comment:
`toUpperCase` can be encapsulated into the module logic so you don't have to know to call it to get the right answer.